single-node/user-data

```diff
@@ -83,7 +83,7 @@ function init_templates {
         mkdir -p $(dirname $TEMPLATE)
         cat << EOF > $TEMPLATE
 [Service]
-ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
```

Comment:
So I should leave the static stuff in `/etc/kubernetes/addons` then? It seems like the word `manifests` is falling out of favor.